src/igr-tests/environ2/run-test.sh

```diff
@@ -14,11 +14,11 @@ RGID=$(id -g)
 for var in USER LOGNAME SHELL; do
     unset $var
 done
-# $UID & $GID are readonly on MacOS
-if [ "$(uname)" != "Darwin" ]; then
-    unset UID
-    unset GID
-fi
+# ignore $UID & $GID unsetting errors
+# Some systems make them readonly (such as MacOS, Void linux)
```

Comment:
It's nothing to do with Void linux, it's caused by the shell that the tests are run with (eg. bash), please adjust the comment appropriately